crates/biome_cli/tests/snapshots/main_commands_format/format_skip_errors_continues_with_valid_files.snap

```diff
@@ -0,0 +1,30 @@
+---
+source: crates/biome_cli/tests/snap_test.rs
+expression: redactor(content)
+---
+## `invalid.js`
+
+```js
+while ) {}
+```
+
+## `valid.js`
+
+```js
+statement();
+
+```
+
+# Emitted Messages
+
+```block
+invalid.js parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  × Skipped file with syntax errors
```

Comment:
This diagnostic is an error, but the CLI returned Ok. I wonder if we should change the severity of this diagnostic. What do you think?

Reply:
yes, since the user should be aware of sytnax errors when the `--skip-errors` option is given, there may be no need to bother with errors here. Maybe it should be info.